js-quirks.md

```diff
@@ -131,6 +133,16 @@ js> [067, 068, 069, 070]
 [55, 68, 69, 56]
 ```
 
+There are also legacy octal escape sequences in strings, and these have
+their own quirks. `'\07' === '\u{7}'`, but `'\08' === '\u{0}8'` since 8
```

Comment:
I think this example might throw people, because of how it is structured. It doesn't make clear that `'\08' === '\u{8}' will be false, this is assumed... One think I realized from learners is even obvious stuff isn't necessarily clear. (like, after you show the learner `+` and `-`, they will know they can use `/`. They don't necessarily!) 

I would move stuff around like this:

For example, `'\07' === '\u{7}'`, but `'\08' !== '\u{8}'`. `8` is not an octal digit, and `\0` followed by `8` or `9` is a legacy octal escape sequence representing the null character. In order to get the above to work, you would need to do `'\08' === '\u{0}8'`. [...]

Reply:
Good call.

Reply:
BTW, you don't have to sell me on "even 'obvious' stuff isn't necessarily clear"—errors like this are just me failing to notice it :)